config/custom_items/items/thomascalloway-oiicoffeecup.json

```diff
@@ -0,0 +1,8 @@
+{
+  "ckey" :            "thomascalloway",
+  "character_name" :  "Thomas Calloway",
+  "item_name" :       "OII coffee cup",
+  "item_desc" :       "This sleek, dark grey coffee cup is branded with the three-letter initialism of the Office of Interstellar Intelligence: OII.",
+  "item_icon_state" : "oii_coffee_cup",
+  "item_path" :       "/obj/item/weapon/reagent_containers/food/drinks/glass2/coffeecup/OII"
```

Comment:
The correct type path is `/obj/item/weapon/reagent_containers/food/drinks/glass2/coffeecup`.

Reply:
You're right! I'm only marginally familiar with GitHub and the process--do I need to delete this pull request and resubmit or can you edit the line?

Reply:
I edited the line, this is flagged as outdated. The new path is exactly as you provided it, omitting /OII from the original.